popSimManu.tex

```diff
@@ -275,12 +275,24 @@ \section*{Results}
 %ACS: the consortium really isn't the point; let's get right to describing what stdpopsim is
 %The first contribution of the PopSim consortium is 
 The \stdpopsim library is a
-a community-maintained collection of empirical genome data and population genetics simulation models.
-Figure \ref{fig:cartoon} shows a graphical representation of the structure of \stdpopsim.
+a community-maintained collection of empirical genome data and population genetics simulation models,
+with a structure depicted in Figure \ref{fig:cartoon}.
 The package centers on a catalog of genomic information and demographic models
 for a growing list of species (Fig.~\ref{fig:cartoon}A), 
 and software resources to facilitate efficient simulations (Fig.~\ref{fig:cartoon}B-C).
 
+Given the genome data and simulation model descriptions defined within the
+library, it is straightforward to run standardized simulations
+across a range of organisms. \Stdpopsim has a Python API and a user-friendly
+command line interface, allowing users with minimal experience direct access to
+state-of-the-art simulations. Simulations are output in the ``tree sequence''
+format \citep{kelleher2016efficient,kelleher2018efficient,kelleher2019inferring}, which
+contains complete genealogical information about the simulated samples, is
+extremely compact, and can be processed efficiently using the \tskit library
+\citep{kelleher2016efficient,kelleher2018efficient}.
+The tree sequence format could also be converted
+to other formats (e.g., VCF) by the user if desired.
+ 
```

Comment:
I moved this up from below, since it seemed odd to have the overview paragraph after the catalog. I think @gphocs-dev moved it down there (but along with other stuff); feel free to disagree?

Reply:
This looks good here